chemprop/v2/data/utils.py

```diff
@@ -1,66 +1,168 @@
 from collections import defaultdict
 from typing import Sequence
+import itertools
 
 import numpy as np
-from rdkit.Chem.Scaffolds import MurckoScaffold
 
 
 from chemprop.v2.data.datapoints import MoleculeDatapoint
+from chemprop.v2.data.datasets import MoleculeDataset
+
+from astartes import train_val_test_split, train_test_split
+from astartes.molecules import train_val_test_split_molecules, train_test_split_molecules
+
+import copy
+from logging import Logger
+import pickle
+from random import Random
+from typing import Tuple
+import os
+
+from chemprop.v2.data.utils import log_scaffold_stats
```

Comment:
Could you combine/organize the imports? e.g., instead of having `from typing import Tuple` and `from typing import Sequence`, combine them as `from typing import Tuple, Sequence`, and group imports from `chemprop.v2` together.